### DISCUSSION TITLE
empty windows in hyprland /kitty on archlinux

### DISCUSSION THREAD

**Discussion by tuxarch** [KRTirtho/spotube#652] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

empty window

### Expected Behavior

I would like to see the interface.
![screenshot_fullscreen_2023-08-24_13-34-10](https://github.com/KRTirtho/spotube/assets/17531010/af95be12-ad73-4ef4-abe8-93f5d5578500)

### Steps to reproduce

Launched the application.
Launched the application from the console.
Run aapimage.

### Operating System

Linux

### Spotube version

3.1.0-1

### Installation source

Arch User Repository (AUR)

### Additional information

package:media_kit_libs_linux registered.
flutter: media_kit: WARNING: package:media_kit_native_event_loop not found.
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: MissingPlatformDirectoryException(Unable to get application documents directory)
#0      getApplicationDocumentsDirectory (package:path_provider/path_provider.dart:122)
<asynchronous suspension>
#1      HiveX.initFlutter (package:hive_flutter/src/hive_extensions.dart:12)
<asynchronous suspension>
#2      QueryClient.initialize (package:fl_query/src/core/client.dart:527)
<asynchronous suspension>
#3      main (package:spotube/main.dart:105)
<asynchronous suspension>

**Comment by KRTirtho** [comment]
- Make sure you've [xdg_user_dirs](https://wiki.archlinux.org/title/XDG_user_directories) (A freedesktop standard. Must be available by default on any desktop linux)
- Also you must have the folder `~/Documents` created (basic directories expected to be created by default on any desktop/server linux)
